inexor/engine/blend.cpp

```diff
@@ -586,7 +586,7 @@ void addblendbrush(const char *name, const char *imgname)
     if(!loadimage(imgname, s)) { spdlog::get("global")->error() << "could not load blend brush image " << imgname; return; }
     if(max(s.w, s.h) > (1<<12))
     {
-        spdlog::get("global")->error("blend brush image size exceeded {}x{} pixels: {}", (1<<12), (1<<12), imgname);
+        spdlog::get("global")->error("blend brush image size exceeded {0}x{1} pixels: {2}", (1<<12), (1<<12), imgname);
```

Comment:
iirc these could have stayed ( https://github.com/fmtlib/fmt (used internally by our logger) does a good job and sets them in case they are left blank)
but maybe its easier to read this way with numbers